examples/src/main/scala/eu/neverblink/jelly/examples/PekkoStreamsEncoderSource.scala

```diff
@@ -65,7 +77,7 @@ object PekkoStreamsEncoderSource extends shared.ScalaExample:
 
     // Here we stream this is as a GRAPHS stream (physical type)
     // You can also use .datasetAsQuads to stream as QUADS
-    val encodedDatasetFuture = RdfSource.builder
+    val encodedDatasetFuture = RdfSource.builder[Model, Dataset, Node, Triple, Quad]()
```

Comment:
Again, are the explicit type params really needed?

Reply:
yes, they are

Reply:
i havent found a good way to direct rdfsource builder to infer the dataset adapter from context without them